cli/src/uploadToApi.ts

```diff
@@ -27,6 +27,13 @@ export const uploadToApi = async ({
   dryRun,
   upload,
 }: UploadToApiArgs): Promise<void> => {
+  console.log(
+    upload && !dryRun ? 'Uploading to Embrace API' : 'Dry run, skipping upload'
+  );
+  if (dryRun || !upload) {
+    return;
+  }
+
   // prepare the body for the API request as a gzipped JSON object
   const body = new Blob([
```

Comment:
Moved the creation of the body after checking if we have to upload, otherwise this is being done for no reason